inst/extdata/meta/colour_scales.csv

```diff
@@ -0,0 +1,8 @@
+country,indicator,colour,min,max,invert_scale
+Malawi,current_art,interpolateViridis,0,80000,FALSE
+Malawi,prevalence,interpolateMagma,0,0.5,TRUE
+Malawi,art_coverage,interpolateViridis,0,1,FALSE
+Malawi,prev,interpolateMagma,0,1,TRUE
+Malawi,artcov,interpolateViridis,0,1,FALSE
```

Comment:
instead of having an invert_scale you could just allow max < min

Reply:
Spoke to Emma to see what would be easier for them to use. She says they already have some logic that relies on max < min so I think they would prefer having a boolean flag for inversion.

Reply:
ok, in that case you should enforce min < max in a test here (you could alternatively allow min > max and then in your wrapper function produce the format Emma expects).  But there is a degree of freedom worth removing here